crates/nu-engine/src/nu_variable.rs

```diff
@@ -25,7 +25,13 @@ pub struct NuVariable {
 
 impl LazyRecord for NuVariable {
     fn column_names(&self) -> Vec<&'static str> {
-        let mut cols = vec!["config-path", "env-path", "history-path", "loginshell-path"];
+        let mut cols = vec![
+            "default-config-dir",
```

Comment:
can we change this to `default-config-path` so we stay consisten with `-path`?

Reply:
`config-path` points to the config file while the new path points at a directory. That's why originally suggested -dir, not -path name.

Reply:
ok, it just looked out of place where everything else is path, but i see the distinction now. thx.